rules.neon

```diff
@@ -29,6 +29,7 @@ parameters:
 		strictCalls: %strictRules.allRules%
 		switchConditionsMatchingType: %strictRules.allRules%
 		noVariableVariables: %strictRules.allRules%
+		strictArrayFilter: %strictRules.allRules%
```

Comment:
See how disallowedLooseComparison is set up, this needs to be combined with bleeding edge